src/kirby/components/chart/options/timeseries.ts

```diff
@@ -16,6 +16,9 @@ export const TimeSeriesOptions: Options = {
     margin: 0,
     marginTop: 48,
   },
+  accessibility: {
```

Comment:
This looks bad (for accessibility)... won't people with sight disabilities have a problem figuring out what the graph is (based on this option)?